lib/index.js

```diff
@@ -3,9 +3,6 @@
  */
 'use strict';
 
-const tls = require('tls');
-tls.DEFAULT_ECDH_CURVE = 'auto';
-
```

Comment:
Is this removed because it's elsewhere or because we now just require node 10?  If the later should bump the packages engine field and add changelog comment about it.

Reply:
According to the `history` here, it was defaulted to `auto` in 10.0.0.

https://nodejs.org/docs/latest-v11.x/api/tls.html#tls_tls_default_ecdh_curve

Reply:
bumped the engine.